ui/src/message_popup/playtest/danson_experience_page2.jsx

```diff
@@ -0,0 +1,181 @@
+
+  contextTypes: {
+    auth: React.PropTypes.object.isRequired
+  },
+
+  getInitialState() {
+    const contextEmail = this.context.auth.userProfile.email;
+    const email = contextEmail === "unknown@mit.edu" ? '' : contextEmail;
+
+    return {
+      email,
+      questions: null,
+      sessionId: uuid.v4()
+    };
+  },
+
+  onStart(email) {
+    const questions = InsubordinationScenarios.questions();
+    this.setState({
+      email,
+      questions
+    });
+  },
+
+  onResetSession() {
+    this.setState(this.getInitialState());
+  },
+
+  onLogMessage(type, response:ResponseT) {
+    const {email, sessionId} = this.state;
+    Api.logEvidence(type, {
+      ...response,
+      sessionId,
+      email,
+      name: email,
+      clientTimestampMs: new Date().getTime(),
+    });
+  },
+
+  render() {
+    return (
+      <SessionFrame onResetSession={this.onResetSession}>
+        {this.renderContent()}
+      </SessionFrame>
+    );
+  },
+
+  renderContent() {
+    const {questions} = this.state;
+    if (!questions) return this.renderIntro();
+
+    return <DansonLinearSession
+      questions={questions}
+      questionEl={this.renderQuestionEl}
+      summaryEl={this.renderSummaryEl}
+      onLogMessage={this.onLogMessage}
+    />;
+  },
+
+  renderIntro() {
+    return (
+      <IntroWithEmail defaultEmail={this.state.email} onDone={this.onStart}>
+        <div>
+            <p>Welcome!</p>
+            <p>You will go through a set of scenarios that simulate the conversation between you and Mrs. Danson.</p>
+            <p>You need to provide an audio response to each prompt. Please respond as quickly as possible (like you would do in a real conversation).</p>
+            <p>This may feel uncomfortable at first, but it's better to feel uncomfortable here than with a real parent.</p>
+        </div>
+      </IntroWithEmail>
+    );
+  },
+
+  // Only ask for audio on questions with choices, otherwise let them continue
+  renderQuestionEl(question:QuestionT, onLog, onResponseSubmitted) {
+    return <RecordThenClassifyQuestion
+        key={question.id}
+        question={question}
+        onLogMessage={onLog}
+        onResponseSubmitted={onResponseSubmitted}
+        forceResponse={true}
+      />;
+  },
+
+  renderSummaryEl(questions:[QuestionT], responses:[ResponseT]) {
+    return (
+      <div className="done">
+        <VelocityTransitionGroup enter={{animation: "slideDown"}} leave={{animation: "slideUp"}} runOnMount={true}>
+          <div style={styles.doneTitle}>
+            <p style={styles.paragraph}>You've finished the simulation.</p>
+            <p style={styles.paragraph}><strong>Do not close this page</strong>. You will need it for the reflection.</p>
+            <p style={styles.paragraph}>Please return to the form for the post-simulation reflection.</p>
+          </div>
+          <p style={styles.summaryTitle}>Summary</p>
+          <Divider />
+
+          {responses.map((response, i) => 
```

Comment:
This might be a good place to look at refactoring `AudioResponseSummary` after to split out the code that shows the summary for a list of question texts and audio URLs.